docs2/site/docs/migrations/migration5.md

```diff
@@ -344,6 +344,77 @@ Field<StringGraphType>("myField", resolve: context =>
 });
 ```
 
+### 12. The `ExecutionStrategy` selected for an operation can be configured through `IGraphQLBuilder`
+
+Previously, in order to change the execution strategy for a specific operation -- for instance,
+using a serial execution strategy for 'query' operation types -- required creating a custom
+document executer and overriding the `SelectExecutionStrategy` method.
+
+Now, for DI configurations, you can call the `.AddExecutionStrategy<T>(OperationType)` method to
+provide this configuration without overriding the method. See below for an example.
+
+```csharp
+// === GraphQL.NET v4 ===
+public class SerialDocumentExecuter : DocumentExecuter
+{
+    public SerialDocumentExecuter(IDocumentBuilder documentBuilder, IDocumentValidator documentValidator, IComplexityAnalyzer complexityAnalyzer, IDocumentCache documentCache, IEnumerable<IConfigureExecutionOptions> configurations)
+        : base(documentBuilder, documentValidator, complexityAnalyzer, documentCache, configurations)
+    {
+    }
+
+    protected override IExecutionStrategy SelectExecutionStrategy(ExecutionContext context)
+    {
+        return context.Operation.Operation switch
+        {
+            OperationType.Query => SerialExecutionStrategy.Instance,
+            _ => base.SelectExecutionStrategy(context)
+        };
+    }
+}
+
+// within Startup.cs
+services.AddGraphQL()
+    .AddSystemTextJson()
+    .AddSchema<StarWarsSchema>()
+    .AddDocumentExecuter<SerialDocumentExecuter>();
+
+
+// === GraphQL.NET v5 ===
+
+// within Startup.cs
+services.AddGraphQL(builder => builder
+    .AddSystemTextJson()
+    .AddSchema<StarWarsSchema>()
+    .AddExecutionStrategy<SerialExecutionStrategy>(OperationType.Query));
+```
+
+You can also register your own implementation of `IExecutionStrategySelector` which can inspect the
+`ExecutionContext` to make additional decisions before selecting an execution strategy.
+
+```csharp
+public class MyExecutionStrategySelector : IExecutionStrategySelector
+{
+    public virtual IExecutionStrategy Select(ExecutionContext context)
+    {
+        return context.Operation.Operation switch
```

Comment:
Maybe show example with "force-serial" flag in `UserContext` here to reveal the meaning of _additional decisions_ ?